src/js/GlobalConstants.js

```diff
@@ -14,7 +14,7 @@ const globalConstants = {
     API: process.env.USASPENDING_API,
     LOCAL: false,
     PERF_LOG: false,
-    MAPBOX_TOKEN: 'pk.eyJ1IjoidXNhc3BlbmRpbmciLCJhIjoiY2l6ZnZjcmh0MDBtbDMybWt6NDR4cjR6ZSJ9.zsCqjJgrMDOA-i1RcCvGvg',
+    MAPBOX_TOKEN: process.env.MAPBOX_TOKEN,
```

Comment:
ha i was wondering about this before but i figured i missed something from last week